### PR TITLE
CP Automated Test - LPS-187698 | master

### PR DESCRIPTION
**Pull request by heiditrindade**
[LPS-187698](https://liferay.atlassian.net/browse/LPS-187698)

### CODE REVIEW

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2DeactivateKeyBeforeSelectingPortal.testcase [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+		task ("When the user selects a project in Customer Portal and chooses to Deactivate") {
+			CustomerPortal.enterOnProject(
+				status = ${statusActive},
+				title = ${testAccount});
+
+			CP2NavigationMenu.goToProductActivation();
+
+			CP2NavigationMenu.clickOnProductButton(product = "Portal");
+		}
```
Comment (styller1) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2DeactivateKeyBeforeSelectingPortal.testcase:60]: In this task it says that he selects deactivate, looking through the macros, in this task only goes to the Portal product, is that right?

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2DeactivateKeyBeforeSelectingPortal.testcase [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+		task ("And go to generate a new key") {
+			CP2ActivationKeys.generateNewKey();
+		}
+
```
Comment (styller1) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2DeactivateKeyBeforeSelectingPortal.testcase:65]: This task could be part of the previous one" When the user selects a project in Customer Portal and chooses to Deactivate", what do you think?

**Comment by heiditrindade** [comment]
ci:test:sf

**Comment by heiditrindade** [comment]
ci:test:stable

**Comment by heiditrindade** [comment]
ci:test:sf

**Comment by heiditrindade** [comment]
ci:test:stable

**Comment by heiditrindade** [comment]
SF is failing due to some error. The fix is already merged, but BChan didn’t publish the new version of SF.

**Comment by styller1** [comment]
ci:test:sf

**Comment by styller1** [comment]
ci:forward

**Pull request closed without merging**